sql/npc_list.sql

```diff
@@ -28546,7 +28546,9 @@ INSERT INTO `npc_list` VALUES (17744216,'Curio_Vendor_Moogle','Curio Vendor Moog
 -- NC: INSERT INTO `npc_list` VALUES (17744233,'NOT_CAPTURED','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
 -- NC: INSERT INTO `npc_list` VALUES (17744234,'NOT_CAPTURED','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
 -- NC: INSERT INTO `npc_list` VALUES (17744235,'NOT_CAPTURED','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17744236,'AMAN_Liaison','A.M.A.N. Liaison',64,68.640,8.500,-221.240,1,50,50,0,0,1,0,3,0x01000C0277106720683066406850006000700000,0,'SOA',0);
+INSERT INTO `npc_list` VALUES (17744236,'AMAN_Liaison','A.M.A.N. Liaison',64,68.640,8.500,-221.240,21,50,50,0,0,1,0,3,0x01000C0277106720683066406850006000700000,0,'SOA',0);
+INSERT INTO `npc_list` VALUES (17744237,'blank','     ',116,-175.167,-6.000,-17.574,1,50,50,0,0,1,0,2051,0x0000AE0900000000000000000000000000000000,130,'SOA',0);
+INSERT INTO `npc_list` VALUES (17744238,'Tales_Beginning','Tales\' Beginning',116,-175.167,-6.000,-17.574,1,50,50,0,17,1,0,3,0x0000810900000000000000000000000000000000,130,'SOA',0);
```

Comment:
Need to be very careful with tales, because they're only visible when a player has skipped.

Reply:
I captured it using a brand new char without ever zoning out of starting zone. So this is the absolute default state.